game/lib/global_leaderboard/podium_widgets.dart

```diff
@@ -3,95 +3,101 @@ import 'package:flutter_svg/flutter_svg.dart';
 
 /**
  * This file contains the 6 possible podium widgets to appear in the leaderboard 
- * page. For each podium position (1,2,3) there is a red and yellow widget,
- * which is yellow if that is the user's current spot and red otherwise.
+ * page. For each podium position (FirstPodium, SecndPodium, ThridPodium) there is a red and orange asset svg variation,
+ * which is orange if that is the user's current spot and red otherwise.
  */
-class FirstPodiumRed extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Center(
-        child: Container(
-      width: 106,
-      height: 112,
-      child: SvgPicture.asset(
-        'assets/icons/red1podium.svg',
-        semanticsLabel: '1st Red Podium',
-      ),
-    ));
-  }
-}
 
-class FirstPodiumYellow extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Center(
-        child: Container(
-      width: 106,
-      height: 112,
-      child: SvgPicture.asset(
-        'assets/icons/yellow1podium.svg',
-        semanticsLabel: '1st Yellow Podium',
-      ),
-    ));
-  }
-}
+var pointsStyle = TextStyle(
```

Comment:
We really should have a file that exports all the styles. I'm not sure if we want to do this now, but it would help a lot with all the new design changes that are happening to have a design system in the flutter code

Reply:
I think this comment may belong in a proposal rather than on this PR specifically.